### PR TITLE
feat: add functions to dynamically generate news and videos json files

### LINKED ISSUE #239: Make news.json and videos.json files not static
https://github.com/ipfs/ipfs-blog/pull/237 adds static files `news.json` and `videos.json` as a stopgap until we can build a dynamic version that pulls the latest items of the news and videos content types. The static files are used to feed the "latest" section on the bottom of ipfs.io.

This issue is for creating a dynamic version of each of these two files.

cc @zebateira

### PR DESCRIPTION
**Pull request by jdiogopeixoto**
- Adds two scripts to dynamically generate the `news.json` and `videos.json` files

Closes #239

### CODE REVIEW

**Review by jessicaschilling — APPROVED** [review]
I'm good with this if @zebateira is!

**Comment by zebateira** [comment]
@jdiogopeixoto could we create single entry script for all the three to simply the logic on the npm script? Something like:

- entry point: `scripts/data/index.js` so on the npm script we just need to run `./scripts/data`
- move the three script files to `scripts/data`

sounds good? it's just so we get some organization going.

**Comment by jdiogopeixoto** [comment]
@zebateira Sure, I will do that 👍

File: scripts/data/latest-news.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
```
Comment (zebateira) [inline scripts/data/latest-news.js:1]: We can remove this now

```suggestion
```

File: scripts/data/latest-videos.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
```
Comment (zebateira) [inline scripts/data/latest-videos.js:1]:
```suggestion
```

**Review by zebateira — CHANGES_REQUESTED** [review]

**Review by zebateira — APPROVED** [review]
LGTM! 🚀

**Pull request merged**